scrabble.js

```diff
@@ -1,16 +1,146 @@
+const LETTER_VALUES = {
+  'A': 1,
+  'E': 1,
+  'I': 1,
+  'O': 1,
+  'U': 1,
+  'L': 1,
+  'N': 1,
+  'R': 1,
+  'S': 1,
+  'T': 1,
+  'D': 2,
+  'G': 2,
+  'B': 3,
+  'C': 3,
+  'M': 3,
+  'P': 3,
+  'F': 4,
+  'H': 4,
+  'V': 4,
+  'W': 4,
+  'Y': 4,
+  'K': 5,
+  'J': 8,
+  'X': 8,
+  'Q': 10,
+  'Z': 10
+};
+const BONUS = 50;
 
 const Scrabble = {
   score(word) {
+    if (!word || typeof word !== 'string' || word.length > 7) {
+      throw new Error('Invalid input');
+    }
```

Comment:
I like that you're throwing full instances of `Error` here. Throwing raw strings is tempting, but you don't get a full stack trace.